### DISCUSSION TITLE
How to print AUX Cases?

### DISCUSSION THREAD

**Discussion by stefanpausch** [pmgducati/Blue-Retro-AIO-Units#33] [Q&A] [answered]
What is the best way (regarding orientation) to print the aux cases. Is support needed?

**Comment by pmgducati** [comment, marked as answer]
I dont have a 3D Printer (Yet) and have been using Shapeways to print the housings.  I would say that if you are printing them at home, I would print the bottoms right side up, there will be some support on the notch up on the System side.  For the top you might need to manually add some support to the bottom so the snap fit arms dont break in the cleaning process

**Reply by stefanpausch** [reply]
I will run some test prints and add information to this topic whenever i figured it out :)

**Reply by stefanpausch** [reply]
@pmgducati : Is it possible to get a AUX Design which has no raised back? (like the main design). - that would make printing much easier and make it more level with the main adapter (otherwise cable weight always pulls back down and put more stress/force on the connection)

By leveling front and back on one plane it would also make the finish of the print much nicer :)

**Reply by pmgducati** [reply]
I can look into that for the future, right now I am working through PCBs for stand alone versions.   When they are wrapped up I can look into this.  Feel free to make an issue for it.

In regards to the cable stress from due to the housing design.  Unless there is absolutely no slack for the wires in the housing this is not an issue.  Even if it was an issue making a flat bottom would not change that fact. 

Side note make sure to put some tiny zip ties around each controller cable in the housing. This will make sure that the cables cannot be pulled out of the housing.

**Reply by GrumpyGopher** [reply]
I was wondering if you are doing stand alone versions for all the systems?  I'm hoping Neo Geo Parallel 1P OD might be one of them.  
thanks

**Reply by pmgducati** [reply]
@tr12345 I created a standalone version in the standalone repo, you have fine the Issue/link to the branch here:

https://github.com/pmgducati/Blue-Retro-Single-Console-Units/issues/2

If you do end up making one please post your findings,  I will then merge it to the main branch

**Reply by GrumpyGopher** [reply]
Thank you for this, if I do end up building it I will post my findings.
thanks

**Reply by GrumpyGopher** [reply]
@stefanpausch I worked up flat versions of the the 1P, 2P, 4P, and Genesis Multitap STL files.  I've only had time to test printing the Multitap.
[Aux Bottom Flat.zip](https://github.com/pmgducati/Blue-Retro-AIO-Units/files/12532834/Aux.Bottom.Flat.zip)
